app/src/main/res/layout/fragment_home.xml

```diff
@@ -1,121 +1,115 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.swiperefreshlayout.widget.SwipeRefreshLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I really don't like that we have whole screen scrollable including Toolbar. That's not that case we would need to hide toolbar and if we do we would need to do it more properly. 
Anyways, let's keep it for now